rebar.config

```diff
@@ -138,12 +138,14 @@
       {ref, "eb3cded5b050edd82cf8653f8c850c6c9890f732"}}},
   {pc, {git, "https://github.com/blt/port_compiler.git", {ref, "c2f3fb1"}}},
   {coveralls, {git, "https://github.com/michalwski/coveralls-erl.git", {ref, "f124a62"}}},
-  {provider_asn1, {git, "https://github.com/knusbaum/provider_asn1.git", {ref, "29f7850"}}}
+  {provider_asn1, {git, "https://github.com/knusbaum/provider_asn1.git", {ref, "29f7850"}}},
+  {rebar3_codecov, {git, "https://github.com/zofpolkowska/rebar3_codecov.git", {branch, "master"}}}
```

Comment:
I suggest to freeze the version when the plugin is finished. Rebar plugins doesn't go to rebar.lock so currently the newest code will be pulled when rebar needs to download the plugin.